lib/src/charts/widgets/bar_chart.dart

```diff
@@ -67,10 +73,11 @@ class _BarChartState extends State<BarChart> with TickerProviderStateMixin {
       targetData: [widget.data],
       animateOnLoad: widget.animateOnLoad,
       autoAnimate: widget.autoAnimate,
-      animation: AnimationController(
-        duration: const Duration(seconds: 1),
-        vsync: this,
-      ),
+      animation: widget.animation ??
+          AnimationController(
+            duration: widget.animDuration ?? const Duration(milliseconds: 500),
```

Comment:
And remove from here
```suggestion
            duration: widget.animDuration,
```